bin/tictactoe

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
```

Comment:
Great bin file, short length is ideal. However, we would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.